src/Backend/Modules/ContentBlocks/Domain/ContentBlock/ContentBlockStatus.php

```diff
@@ -1,68 +0,0 @@
-<?php
```

Comment:
can you rebase and actually remove these files in the same commit as you create the new class so that git sees it as a move?